cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -124,12 +126,18 @@ def get_blacklist_of_fields(cls, course_key):
         # If the ENABLE_PROCTORING_PROVIDER_OVERRIDES waffle flag is not enabled,
         # do not show "Proctoring Configuration" in Studio Advanced Settings.
         if not ENABLE_PROCTORING_PROVIDER_OVERRIDES.is_enabled(course_key):
-            black_list.append('proctoring_configuration')
+            black_list.append('proctoring_provider')
 
         # Do not show "Course Visibility For Unenrolled Learners" in Studio Advanced Settings
         # if the enable_anonymous_access flag is not enabled
         if not COURSE_ENABLE_UNENROLLED_ACCESS_FLAG.is_enabled(course_key=course_key):
             black_list.append('course_visibility')
+
+        # Do not show "Create Zendesk Tickets For Suspicious Proctored Exam Attempts" in
+        # Studio Advanced Settings if the user is not edX staff.
+        if not GlobalStaff().has_user(get_current_user()):
```

Comment:
This assumes that if you are accessing a course in Studio and are not global staff (i.e. edX admin or edX staff), you must be a course team member. Is that a fair assessment?

Reply:
It's fair. Unauthenticated users won't be able to get here, anyway.